src/shared/lib/hooks/useAudioPlayer.ts

```diff
@@ -10,7 +10,7 @@ const SUBSCRIPTION_DURATION = 500;
 const useAudioPlayer = () => {
   const audioRecorderPlayer = useRef(new AudioRecorderPlayer());
   const [isPlaying, setIsPlaying] = useState(false);
-  const [isLoading, setIsLoading] = useState(false);
+  const [isLoading, setIsLoading] = useState<null | boolean>(null);
```

Comment:
What if we pass `true` instead as an initial value? This way, you will not have to do `setIsLoading(true)` at the start of the `play` function at all.